media-gfx/kuickshow/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="project">
+	<email>kde@gentoo.org</email>
```

Comment:
You either need @gentoo/kde to want it, or you need to add yourself and proxy-maint instead. That said, considering all that has been said, I seriously doubt @gentoo/proxy-maint is going to take this unless you really volunteer to keep the package alive (including porting to Qt5).